recipe/meta.yaml

```diff
@@ -19,17 +19,23 @@ build:
 requirements:
   host:
     - python
+    - setuptools
     - pip
   run:
     - python
 
 test:
   commands:
+    - pip check
     - futurize -h
     - pasteurize -h
-
   imports:
     - future
+    - libfuturize
+    - libpasteurize
```

Comment:
I'm not sure these `lib*` modules are meant to be directly imported by the user; they seem more like support for the `futurize` and `pasteurize` scripts. If that's the case, we should just remove them from the `imports` tests.